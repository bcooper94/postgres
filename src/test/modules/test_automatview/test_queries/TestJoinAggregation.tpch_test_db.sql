SET automatview.training_sample_count = 1;

select
	l_orderkey,
	sum(l_extendedprice),
	o_orderdate,
	o_shippriority
from
	customer join orders on c_custkey = o_custkey
	join lineitem on l_orderkey = o_orderkey
group by
	l_orderkey,
	o_orderdate,
	o_shippriority;

select
	l_orderkey,
	sum(l_extendedprice),
	o_orderdate,
	o_shippriority
from
	customer join orders on c_custkey = o_custkey
	join lineitem on l_orderkey = o_orderkey
group by
	l_orderkey,
	o_orderdate,
	o_shippriority;
