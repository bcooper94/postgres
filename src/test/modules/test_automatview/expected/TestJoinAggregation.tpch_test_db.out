1
SELECT Matview.lineitem_l_orderkey AS Matview_lineitem_l_orderkey, Matview.sum_lineitem_l_extendedprice AS Matview_sum_lineitem_l_extendedprice, Matview.orders_o_orderdate AS Matview_orders_o_orderdate, Matview.orders_o_shippriority AS Matview_orders_o_shippriority FROM Matview